src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/http_client.html.twig

```diff
@@ -61,7 +72,7 @@
                                             <span class="label">{{ trace.method }}</span>
                                         </th>
                                         <th class="full-width">
-                                            {{ trace.url }}
+                                            <a href="{{ trace.url }}">{{ trace.url }}</a>
```

Comment:
I would make it a link only when the method is GET.

Reply:
shoudl be done

Reply:
Can you please revert this change?
A copy-as-curl button would be a better alternative. Meanwhile, no need to add this link.